scripts/build_base_energy_totals.py

```diff
@@ -113,12 +132,16 @@ def calc_sector(sector):
             sectors_dfs[sector] = df_sector.copy()
 
             if sector == "consumption by households":
+                if snakemake.config["sector"]["coal"]["shift_to_elec"]:
```

Comment:
It seems to be that here as well operations may be similar and may be aggregated.
For example, if we define a dictionary like:
```
d_config = {
"consumption by households": ["Electricity"],
"services": ["oil", "biomass", ...]
...
}
```
And we also standardize the oil_fuels/biomass_fuels etc in an automated way, the script can be significantly improved.
Probably worth to keep in mind this and add an issue about it?
Revising this script can be quite heavy and I don't think it is a priority

Reply:
I think an issue here will be useful. Again, I think this is an interesting task for early users especially with software background

Reply:
Added to issue: https://github.com/pypsa-meets-earth/pypsa-earth-sec/issues/358